### PR TITLE
Rewording points in 3.10

### PR DESCRIPTION
**Pull request by kerimoyle**
Addresses #170, #171, #172, #173 

This is a straw-man for some rewording of the map variables section in chapter 3.

### CODE REVIEW

File: src/reference/sectionC_interpretation.inc [hunk 1]
```diff
@@ -373,19 +367,25 @@ Interpretation of ``map_variables`` elements
...
 
-#.  The :code:`variable` elements in a CellML model SHALL be treated as belonging to a single "connected variable set".
-    Each set of connected variables is the set of all :code:`variable` elements for which the corresponding nodes in the variable equivalence network form a connected subgraph.
-    Each set of connected variables represents one variable in the underlying mathematical model.
+#.  For each :code:`map_variables` element present in the CellML model, the variable unit reduction (see :ref:`Effect of units on variables <specC_effect_of_units_on_variables>` ) of variable *A* MUST have an identical set of tuples to the variable unit reduction of variable *B*.
+    Two sets of tuples SHALL be considered identical if all of the tuples from each set are present in the other, or if both sets are empty.
+    Two tuples are considered identical if and only if both the name and exponent value of each tuple are equivalent.
+
+#.  Tuples differing by a multiplying factor in their unit reduction MUST be taken into account when interpreting the numerical values of the variables (see :ref:`Interpretation of units<specC_interpretation_of_units>`).
```
Comment (MichaelClerx) [inline src/reference/sectionC_interpretation.inc:384]: Is this line not better left for the interpretation section?

File: src/reference/sectionC_interpretation.inc [hunk 2]
```diff
@@ -373,19 +367,25 @@ Interpretation of ``map_variables`` elements
...
+
+#.  The :code:`variable` elements in a CellML model SHALL be treated as belonging to a set, connected by the relationships specified in the :code:`math` elements of the model.
```
Comment (MichaelClerx) [inline src/reference/sectionC_interpretation.inc:386]: Again, this bit goes in the interpretation part

**Comment by MichaelClerx** [comment]
If we go with this (which I'd like), a lot of 3.9 can go. Who needs all these sets!

**Comment by kerimoyle** [comment]
Or maybe just a simple glossary at the beginning of chapter 3? then we can define the element interpretation in a sensible order independent of definitions?

**Comment by MichaelClerx** [comment]
Hmmmm! No I like the idea of the interpretation saying "this is what the syntax means", so not quite a glossary, just some actual rules about what encapsulation _means_ ?

**Comment by kerimoyle** [comment]
Haha, no I mean a glossary that explains some convenient shorthand for the interpretation section, eg: test value = value of the variable referenced by the reset element’s `test_variable` attribute, etc

**Comment by kerimoyle** [comment]
Though what encapsulation (or anything) means should (haven't checked ...) be explained in the informative spec as an opener?

**Comment by MichaelClerx** [comment]
For the avoidance of doubt, what we currently have is correct so I'm not too upset if it stays in, but I think it could be a whole lot simpler, e.g.

> - Connections can only be made between variables of components that are either siblings in the encapsulation hierarchy, or have a parent-child relationship.
> - Variables of components that are siblings in the encapsulation hierarchy are connected through the public interface.
> - In parent-child relationships, the variable in the parent component uses the private interface, while the variable in the child component uses the public interface.

**Comment by kerimoyle** [comment]
Changes to 3.10.9 as per meeting: https://cellml-specification-dev.readthedocs.io/en/normativerewording_3_10/reference/formal_and_informative/specC10_interpretation_of_map_variables.html

**Comment by kerimoyle** [comment]
This still needs the informative sections to be reordered to match the change in order of 3.10, but I'll sort them out separately.

**Pull request merged**